packages/pds-ember/addon/components/pds/textarea/stories/enabled/invalid.stories.js

```diff
@@ -0,0 +1,43 @@
+import hbs from 'htmlbars-inline-precompile';
+import Parent from './index.stories.js';
+
+export default {
+  ...Parent,
+  title: `${Parent.title} / Invalid`,
+}
+
+export const Blurred = () => ({
+  template: hbs`
+    <Pds::Textarea
+      @invalid={{invalid}}
+    />
+  `
+});
+
+export const Hover = () => ({
+  template: hbs`
+    <Pds::Textarea
+      class="mock-hover"
+      @invalid={{invalid}}
+    />
+  `
+});
+
+export const Focus = () => ({
+  template: hbs`
+    <Pds::Textarea
+      class="mock-focus"
+      @invalid={{invalid}}
+    />
+  `
+});
+
+export const FocusHover = () => ({
+  template: hbs`
+    <Pds::Textarea
+      class="mock-focus mock-hover"
+      @invalid={{invalid}}
+    />
+  `
+});
+FocusHover.storyName = 'Focus + Hover';
```

Comment:
All of these don't render as invalid fwiw - https://deploy-preview-21--hashicorp-structure.netlify.app/?path=/story/components-textarea-enabled-invalid--blurred

Reply:
Maybe because `context.invalid` isn't ever set?

Reply:
I may have missed this. I'll take a look.

Reply:
Ah, I forgot to hard-code `@invalid={{true}}`. Fixed.